testing/dart/image_descriptor_test.dart

```diff
@@ -70,6 +70,19 @@ void main() {
     final Codec codec = await descriptor.instantiateCodec();
     expect(codec.frameCount, 1);
   });
+
+  test('HEIC image', () async {
+    final Uint8List bytes = await readFile('grill_chicken.heic');
+    final ImmutableBuffer buffer = await ImmutableBuffer.fromUint8List(bytes);
+    final ImageDescriptor descriptor = await ImageDescriptor.encoded(buffer);
+
+    expect(descriptor.width, 300);
+    expect(descriptor.height, 400);
+    expect(descriptor.bytesPerPixel, 4);
```

Comment:
Somewhat unrelated thought:  Do we always just create RGBA bytes?  An heic image shouldn't have an alpha channel so this is a waste.

Reply:
We support non RGBA formats, and some formats would definitely have this value be lower - but for this format, that's what Skia is telling us CoreGraphics is saying.